### DISCUSSION TITLE
Content response from NERC Vocabulary Server seems change that makes ArgoNVSReferenceTables fail

### DISCUSSION THREAD

**Discussion by cywhale** [euroargodev/argopy#376] [Q&A]
```
from argopy import ArgoNVSReferenceTables
R = ArgoNVSReferenceTables()
print(R)
# got <argopy.related.reference_tables.ArgoNVSReferenceTables object at 0x7f764b52ee10>
R.search('sensor')
```
DataNotFound: "['https://vocab.nerc.ac.uk/collection/R01/current/?_profile=nvs&_mediatype=application/ld+json', 'https://vocab.nerc.ac.uk/collection/RR2/current/?_profile=nvs&_mediatype=application/ld+json',
#..skip...
'https://vocab.nerc.ac.uk/collection/R40/current/?_profile=nvs&_mediatype=application/ld+json']"

If I use browser to open each of the URL in the elements of this array, it seems some of the URLs got lost, e.g. https://vocab.nerc.ac.uk/collection/R29/current/?_profile=nvs&_mediatype=application/ld+json
But some of them are still alive, e.g. https://vocab.nerc.ac.uk/collection/R13/current/?_profile=nvs&_mediatype=application/ld+json

However, even I just read the R13 table by:
```
R.tbl('R13')
```
Got KeyError: 'altLabel'

It seems the JSON structure of the reponse changes:
```
{
    "@graph": [
        {
            "@id": "http://vocab.nerc.ac.uk/collection/R40/current/PI0107/",
            "pav:authoredOn": "2023-06-22 12:58:04.0",
            "pav:hasCurrentVersion": {
                "@id": "http://vocab.nerc.ac.uk/collection/R40/current/PI0107/1/"
            },
            "dce:identifier": "SDN:R40::PI0107",
            "pav:version": "1",
            "skos:notation": "SDN:R40::PI0107",
            "skos:altLabel": "Juliet HERMES",
            "dc:date": "2023-06-22 12:58:04.0",
            "owl:versionInfo": "1",
            "skos:prefLabel": {
                "@language": "en",
                "@value": "Juliet HERMES"
            },
//skip.....
}            
```
Only 'skos:altLabel' exist.
I'm not familiar with `argopy` usage, but it could work when I last used it two months ago. I found no discussions/issuses here or in https://github.com/nvs-vocabs/ArgoVocabs so I'm not sure what's the possible cause for this problem.

**Comment by gmaze** [comment]
Hi @cywhale 
thanks for reporting this !
I created an issue #377 to investigate
It's weird though, because daily tests are passed on this feature
Currently checking

**Reply by gmaze** [reply]
> It's weird though, because daily tests are passed on this feature

1st day back from vacations, my mind is still rusty !

Tests are passed because they used a mocked NVS server,
So if the server has changed, there's no way for the tests to know it

**Comment by gmaze** [comment]
This should be fixed in the next release ! #378
